api/withdrawal.go

```diff
@@ -156,7 +177,10 @@ func (c *Call) FeeWithdrawal(ctx context.Context, request model.FeeWithdrawalReq
 		SetAuthToken(c.bearerToken).
 		SetBody(request).
 		SetResult(&response).
-		SetHeader("Signature", signature).
+		SetHeaders(map[string]string{
+			"Signature":              signature,
+			model.RequestIDHeaderKey: requestID,
+		}).
```

Comment:
??????????

Reply:
Removed too

Reply:
@wharleyinc my bad. please keep this. I thought you were replacing the signature w the header.